settings.gradle

```diff
@@ -1,2 +1,3 @@
-include 'demo-assethelper', 'demo-sqliteprovider'
-
+include 'demo-assethelper'
+include 'demo-sqliteprovider'
+include 'analyzer'
```

Comment:
Yes, that's because it is a separate module. You can import just the `analyzer`'s build.gradle as a project by itself. 
Not ideal, but helps to link the plugin's sources to the sample projects.

I usually have the two projects open (in two different windows).